### DISCUSSION TITLE
11/21 반재영 블로그 1회차

### DISCUSSION THREAD

**Discussion by baaanjy** [Orm-camp/blog-feedback#38] [동글동글 3기 피드백]
<a href="https://velog.io/@baaanjy/오름캠프-미니프로젝트-회고">https://velog.io/@baaanjy/오름캠프-미니프로젝트-회고</a>
미리프로젝트 회고 글입니다 😀

**Comment by oratio100** [comment]
깔끔하게 잘 쓰인 회고입니다. 굳이 의견을 내자면 나의 구현 기능을 좀 더 자세히 써주시는 것도 좋을 것 같습니다:)

**Comment by mh0223** [comment]
담백하게 프로젝트 경험을 기술하신 게 좋은 것 같습니다! KPT 양식으로 작성해주셔서 앞으로의 시도해볼 점도 나타나서 좋은 것 같아요.

**Comment by kimjigyun** [comment]
새로운것을 써보면서 공부가 많이 되었을것 같아요 ㅎㅎ 저도 열심히 배워보겠습니다

**Comment by ksj0621** [comment]
정말 담백하고 군더더기 없는 회고인 것 같아요! 하지만 그런 점에서 좀 더 재영님의 개인적인 생각이 드러났으면 좋지 않았을까 싶기도 했습니다:)

**Comment by Orm-camp** [comment]
Keep 부분에서 재영님의 훌륭한 마인드를 엿볼 수 있어서 너무 좋은 글이라고 생각합니다!
Try에서도 명확하게 무엇을 더 공부하고 해나가야 하는지 제시하셔서, 모르는 사람이 읽더라도 재영님은 발전 가능성이 매우 높은 사람이라고 인식될 것 같아요.
너무 멋진 셀프 브랜딩의 시작! 계속해서 응원하겠습니다 👍🏻
